src/views/HomeView.vue

```diff
@@ -1,41 +1,66 @@
 <template>
   <div class="home-container">
-    <h1>Schule 4.0 - Haustiere</h1>
-    <p>Wähle ein Haustier um das Spiel zu starten:</p>
-    <select v-model="selectedPet" @change="goToPetStages">
-      <option disabled value="">Wähle ein Haustier</option>
-      <option value="dog">Hund</option>
-    </select>
+    <div class="header-box">
+      <h1>Mein erstes Haustier</h1>
+      <img src="../assets/icon_audio.svg" alt="Audio">
+    </div>
+
+    <p>Bereit, in die Welt der Haustiere einzutauchen?
+    <br>Tippe auf das Tier, das dich am meisten interessiert!</p>
+  </div>
+
+  <div class="pet-cards">
+      <PetCardComponent v-for="(pet, index) in pets" :key="index" :pet="pet" @pet-selected="handlePetSelected" />
   </div>
 </template>
 
 <script setup lang="ts">
-import { ref } from 'vue'
 import { useRouter } from 'vue-router'
+import PetCardComponent from '@/components/PetCardComponent.vue';
+import pets from '@/config/petsConfig';
 
-const selectedPet = ref('')
 const router = useRouter()
 
-const goToPetStages = () => {
-  if (selectedPet.value) {
-    router.push(`/pets/${selectedPet.value}/stages/1`)
+const handlePetSelected = (selectedPet: any) => {
```

Comment:
Avoid using `any` unless absolutely necessary. It bypasses type checking, reducing code safety and clarity.
Instead export the `Pets `interface in` petsConfig.ts` and use it here `selectedPet: Pet`